scss/partials/_cards.scss

```diff
@@ -276,8 +296,13 @@
     .content-block__title {
       @extend .h6;
     }
-  }
 
+    .content-block--blog {
+      @include breakpoint(desktop) {
+        width: fit-content !important;
```

Comment:
Here and above, is it possible to refactor a selector to avoid the use of !important? If it's not, what's the problem we created that requires it?

Reply:
let me resolve most `!important` for the homepage blog. There's another one but it's for the main blog. I'll leave that one for post mvp ticket